app/assets/stylesheets/lifehack_style.scss

```diff
@@ -160,9 +188,70 @@
 }
 
 .add-lifehack {
-  border: 1px solid $green;
+  border: 0.1rem solid $oil;
```

Comment:
Properties should be ordered background-color, border, border-radius, font-size, font-size, width